css-view-transitions-1/Overview.bs

```diff
@@ -1262,43 +1262,16 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. [=Queue a global task=] on the [=DOM manipulation task source=],
 			given |transition|'s [=relevant global object=],
-			to execute the following steps:
-
-				Note: A task is queued here because the texture read back in [=capturing the image=] may be async,
-					although the render steps in the HTML spec act as if it's synchronous.
+			to perform the following steps:
 
 			1. If |transition|'s [=ViewTransition/phase=] is "`done`", then abort these steps.
 
 				Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
-					The [=skip the view transition=] steps [=call the update callback=],
-					ensuring the |transition|'s [=ViewTransition/update callback=] is always called.
-
-			1. [=Call the update callback=] of |transition|.
-
-			1. [=promise/React=] to |transition|'s [=ViewTransition/update callback done promise=]:
-
-				* If the promise does not settle within an implementation-defined timeout, then:
-
-					1. If |transition|'s [=ViewTransition/phase=] is "`done`", then return.
-
-						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
 
-					1. [=Skip the view transition=] |transition| with a "{{TimeoutError}}" {{DOMException}}.
+			1. [=call the update callback=].
 
-				* If the promise was rejected with reason |reason|, then:
-
-					1. If |transition|'s [=ViewTransition/phase=] is "`done`", then return.
-
-						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
-
-					1. [=Mark as handled=] |transition|'s [=ViewTransition/ready promise=].
-
-						Note: |transition|'s [=ViewTransition/update callback done promise=] will provide the {{unhandledrejection}}.
-						This step avoids a duplicate.
-
-					1. [=Skip the view transition=] |transition| with |reason|.
-
-				* If the promise was fulfilled, then [=activate view transition|activate=] |transition|.
+				Note: A task is queued here because the texture read back in [=capturing the image=] may be async,
```

Comment:
Do you mind restoring the position of this note? Nicer to have this next to where we post the task. Otherwise its part of this sub-point which is less clearer.

Reply:
Done